panoptica/utils/input_check_and_conversion/sanity_checker.py

```diff
@@ -0,0 +1,171 @@
+from typing import Any
+import numpy as np
+from pathlib import Path
+from warnings import warn
+
+from panoptica.utils.constants import _Enum_Compare
+from panoptica.utils.input_check_and_conversion.input_data_type_checker import (
+    _InputDataTypeChecker,
+)
+from panoptica.utils.input_check_and_conversion.check_numpy_array import (
+    sanity_checker_numpy_array,
+)
+from panoptica.utils.input_check_and_conversion.check_sitk_image import (
+    sanity_checker_sitk_image,
+)
+from panoptica.utils.input_check_and_conversion.check_torch_image import (
+    sanity_checker_torch_image,
+)
+from panoptica.utils.input_check_and_conversion.check_nibabel_image import (
+    sanity_checker_nibabel_image,
+)
+from panoptica.utils.numpy_utils import _get_smallest_fitting_uint
+
+
+class INPUTDTYPE(_Enum_Compare):
+    NUMPY = _InputDataTypeChecker(
+        supported_file_endings=[".npy", ".npz"],
+        required_package_names=["numpy"],
+        sanity_check_handler=sanity_checker_numpy_array,
+    )
+    SITK = _InputDataTypeChecker(
+        supported_file_endings=[".nii", ".nii.gz", ".mha", ".mhd", ".dcm"],
+        required_package_names=["SimpleITK"],
+        sanity_check_handler=sanity_checker_sitk_image,
+    )
+    TORCH = _InputDataTypeChecker(
+        supported_file_endings=[".pt", ".pth"],
+        required_package_names=["torch"],
+        sanity_check_handler=sanity_checker_torch_image,
+    )
+    NIBABEL = _InputDataTypeChecker(
+        supported_file_endings=[".nii", ".nii.gz"],
+        required_package_names=["nibabel"],
+        sanity_check_handler=sanity_checker_nibabel_image,
+    )
```

Comment:
This is redundant, as all functionality related to nibabel is taken care of by SimpleITK.